tools/longreadsum.cwl

```diff
@@ -30,7 +26,7 @@ inputs:
     type:
       type: enum
       name: input_type
-      symbols: ["bam","f5","fa","fq"]
+      symbols: ["bam","f5","f5s","fa","fq","seqtxt"]
```

Comment:
seqtext? sounds like some old sanger sequencing

Reply:
I've never seen or heard of it before writing the option.